tests/src/Interop/COM/ServerContracts/readme.md

```diff
@@ -0,0 +1,9 @@
+## Server Contracts
+
+This directory contains the API contracts for the testing of .NET COM interop. The contract is defined in C# to the degree that the [TlbExp.exe](https://docs.microsoft.com/en-us/dotnet/framework/tools/tlbexp-exe-type-library-exporter) tool can be used to generate a TLB that can then be used by the Microsoft VC++ compiler to generate the `tlh` and `tli` files. This is a manual process at the moment, but as more support is added to CoreCLR, this may change.
+
+The process to create a TLB and update the native contracts are as follows:
+
+1) Take the `Primitives.cs` file and create a DLL using a SDK project.
```

Comment:
why do you choose cs file as a contract instead of IDL file?

Reply:
Because in this basic scenario it is possible to use TlbExp.exe and validate that scenario as well. We will need to use IDL for a myriad of other scenarios so that is going to happen.